tools/ktx/command_create.cpp

```diff
@@ -1329,6 +1350,16 @@ void CommandCreate::executeCreate() {
                 image->yflip();
             }
 
+            if (options.normalize) {
+                if (target.format().transfer() != KHR_DF_TRANSFER_LINEAR) {
+                    fatal(rc::INVALID_FILE,
+                        "Input file \"{}\" transfer function is not linear. Normalize is only available for linear images. "
```

Comment:
"The transfer function to be applied to the created texture is neither linear nor none. Normalize ..."

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/977/commits/61556ccf343abe68c01e8497f00c9ca2f786170c